docs/source/api/v4/oc_ci_configuration.rst

```diff
@@ -23,22 +23,26 @@
 =======
 Triggers an asynchronous task to update the configuration for the :abbr:`uCDN (Upstream Content Delivery Network)` by adding the request to a queue to be reviewed later. This returns a 202 Accepted status and an endpoint to be used for status updates.
 
+.. note:: Users with an "admin" role will need to provide a "ucdn" query parameter to bypass the need for :abbr:`uCDN (Upstream Content Delivery Network)` information in the :abbr:`JWT (JSON Web Token)` and allow admins to view all :abbr:`CDNi (Content Delivery Network Interconnect)` information.
```

Comment:
There's only one "admin" Role. The way this is set up currently, there's no way to make any Role that has permission to bypass the need for uCDN info. Only users with exactly the "admin" Role, regardless of any Permission set or "Priv Level", will be able to use this functionality. Even the "admin" Role itself can't create a Role that's allowed to do this. If that's intentional, then this is just a semantic change in the docs ("an"->"the"). If you want to declare some set of behavior that requires special Permissions, you should use a Permission.